packages/core/src/components/Form.js

```diff
@@ -68,8 +68,8 @@ export default class Form extends Component {
     let { errors, errorSchema } = mustValidate
       ? this.validate(formData, schema, additionalMetaSchemas, customFormats)
       : {
-          errors: state.errors || [],
-          errorSchema: state.errorSchema || {},
+          errors: (!props.noValidate && state.errors) || [],
```

Comment:
This fix only works for when `noValidate` is true, but it doesn't fix the issue when `liveValidate` is false (#1547). I've went ahead and added a test for that one. Do you have an idea of what might fix that one?

Reply:
The bug for `liveValidate` was very similar to the bug being fixed: `mustValidate` being false, state errors were used and they contained the old `extraErrors`. I updated the code to check for both `liveValidate` and `noValidate` before using state errors.